src/script.js

```diff
@@ -0,0 +1,265 @@
+const roomURL = 'DAILY_ROOM_URL';
+const kahootQuizURL =
```

Comment:
Thought I'd place this at the top since this is one of those things the user might want to modify. And having at the top makes immediately clear what quiz will be loaded when they do start the integration